spec/import/rummager_data_presenter_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'import/rummager_data_presenter'
+
+module Import
+  RSpec.describe RummagerDataPresenter do
+
+    it "presents row data from rummager in a form suitable for insertion in the local sqlite db" do
+
+      row_data = test_data_example
+
+      expected_rows = [
+          ["/vehicle-tax", "organisations", "{/government/organisations/driver-and-vehicle-licensing-agency,/government/organisations/test-org}"],
```

Comment:
those curly braces represent an array in postgresql, should we keep it or simply remove them and comma separate?